src/app/core/pwa-install/pwa-install.component.scss

```diff
@@ -4,6 +4,6 @@
   border-top: solid 1px rgba(0, 0, 0, 0.12);
   border-radius: 0;
   overflow: hidden;
-  background-color: colors.$primary;
   width: 100%;
+  color: white;
```

Comment:
is there a specific reason for making this white?

Reply:
the top toolbar has the same background and white text, so this seems more consistent (although I don't feel good setting it explicitly). At different color schemes the contrast with black text was even worse, also.